programs/adapter-tulip/src/lib.rs

```diff
@@ -660,3 +639,46 @@ pub fn load_remaining_accounts<'info>(
     }
     return accounts;
 }
+
+pub fn account_info_array<'info>(
+    remaining_accounts: &[AccountInfo<'info>],
+    index_array: Vec<usize>,
+) -> [AccountInfo<'info>; 30] {
+    let mut accounts: [AccountInfo<'info>; 30] = [
+        remaining_accounts[0].clone(),
```

Comment:
Try:

```rust
let mut accounts: [AccountInfo<'info>; 30] = [remaining_accounts[0]; 30]
```

Reply:
Can return `&[AccountInfo]` ?

Reply:
it might have this error -> `anchor_lang::prelude::AccountInfo<'info>: std::marker::Copy` is not satisfied

Reply:
Can return `&[AccountInfo]` ?

Reply:
No, can only return a static array.